style.css

```diff
@@ -0,0 +1,166 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@200&family=Syne+Mono&display=swap');
+* {
+    margin: 0;
+    padding: 0;
+    font-family: 'Montserrat', sans-serif;
+    font-weight: bold;
+}
+body{
+    background: #736453;
+    background: linear-gradient(90deg,#736453 0%, #f2f2f2 50%,#736453 100%);
+}
+h1{
+    font-size: 70px;
+    margin-top: 2%;
+}
+.header{
+    text-align: center;
+    color:#261201;
+    font-family: 'Montserrat', sans-serif;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+.flex-container{
+    display: flex;
+    justify-content: center;
+    margin-top: 5%;
+}
+.container {
+    height: 460px;
+    background-color: #F2F2F2;
+    -webkit-box-shadow: 0px 11px 40px 12px rgba(0,0,0,0.78); 
+    box-shadow: 0px 11px 40px 12px rgba(0,0,0,0.78);
+}
+.display{
+    background-color:#F2F2F2;
+    height: 62px;
+    width: 100%;
+    text-align: end;
+    font-size: 45px;
+    color: #404040;
+}
+p{
+    margin:2% ;
+}
+hr{
+    height: 1px;
+    border: none;
+    background-color:  #261201;
+}
+.first-row,
+.second-row,
+.third-row,
+.forth-row,
+.fifth-row,
+.sixth-row{
+    display: flex;
+    width: 100%;
+    height: 65px;
+}
+.Calc__btn{
+    border:0.5px solid #736453;
+    text-decoration: none;
+    color: #261201 ;
+    font-family: 'Montserrat', sans-serif;
+    font-size: 25px;
+    height: 100%;
+    cursor: pointer;
+    outline: none;
+}
+#clear{
+    background-color:#590202 ;
+    color: #F2F2F2;
+    height: 65px;
+    width: 50%;
+    border: none;
+    font-family: 'Montserrat', sans-serif;
+}
+#divide{
+    background-color:#261201;
+    color: #F2F2F2;
+    height: 65px;
+    width: 25%;
+}
+#multiply{
+    background-color:#261201;
+    color: #F2F2F2;
+    height: 65px;
+    width: 25%;
+}
+#seven,
+#eight,
+#nine,
+#four,
+#five,
+#six,
+#one,
+#two,
+#three{
```

Comment:
also here add to all of these elements the same class name and apply the style on it.